packages/generator-single-spa/src/common-templates/typescript/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "devDependencies": {
     "@babel/preset-typescript": "^7.9.0",
-    "eslint-config-ts-react-important-stuff": "^2.0.0",
+    "eslint-config-ts-important-stuff": "^1.0.0",
```

Comment:
We should delete this from generated react projects. See similar code at https://github.com/single-spa/create-single-spa/blob/f043b6d40f9a365ebf35528304a39754ba670073/packages/generator-single-spa/src/react/generator-single-spa-react.js#L65.

To check that it's working, you can go into the test fixtures and verify the package json only has eslint-config-react-ts-important-stuff instead of eslint-config-ts-important-stuff.